pea/formatter.py

```diff
@@ -23,7 +20,7 @@ def __init__(self, *args):
 		type(self).instance = self
 
 	def setOutputStream(self, stream):
-		type(self).stream = stream
+		type(self).stream = colorama.initialise.wrap_stream(stream, True, False, False, True)
```

Comment:
Yeh, maybe. I really just copied them from how they're used in ``colorama.initialise``, they're not really well documented.